SQL/9999-99-99-drop_tables.sql

```diff
@@ -1,3 +1,9 @@
+DROP TABLE IF EXISTS dataquery_run_results;
+DROP TABLE IF EXISTS dataquery_run_queries;
+DROP TABLE IF EXISTS dataquery_shared_queries_rel;
+DROP TABLE IF EXISTS dataquery_pinned_queries_rel;
+DROP TABLE IF EXISTS dataquery_queries;
```

Comment:
there are 7 tables in the schema file and only 5 here

Reply:
So there are.. I had just copied the drop tables that were in the New_patches patch. Should be fixed now.